mpyl_config.example.yml

```diff
@@ -1,15 +1,6 @@
 ---
 vcs:
   changedFilesPath: !ENV ${CHANGED_FILES_PATH}
-  git:
```

Comment:
The Actions workflow runs against the **current** version of the Docker image (which still requires git), not against the changes in this PR. The Docker workflow runs against the changes in this PR and [is very green](https://github.com/Vandebron/gh-mpyl/actions/runs/11777221412) 🟢 🙂  


(I'll see if I can make the other workflow skip in this situation, it can be confusing)

Reply:
Ah you've managed to reproduce the config schema issues we have with the monorepo 😉